modules/text_server_adv/text_server_adv.h

```diff
@@ -151,6 +152,9 @@ class TextServerAdvanced : public TextServerExtension {
 	HashMap<StringName, int32_t> feature_sets;
 	HashMap<int32_t, FeatureInfo> feature_sets_inv;
 
+	SafeNumeric<TextServer::FontLCDSubpixelLayout> lcd_subpixel_layout = SafeNumeric<TextServer::FontLCDSubpixelLayout>(TextServer::FontLCDSubpixelLayout::FONT_LCD_SUBPIXEL_LAYOUT_NONE);
```

Comment:
```suggestion
	SafeNumeric<TextServer::FontLCDSubpixelLayout> lcd_subpixel_layout{ TextServer::FontLCDSubpixelLayout::FONT_LCD_SUBPIXEL_LAYOUT_NONE };
```
Seems to be preferred syntax elsewhere

Reply:
i see,, i kinda assume c++11 syntax will be rejected if i use it, bc auto and lambdas aren't allowed, also srry for the commit spam , couldn't figure out how to install the precommit git hooks

Reply:
You can also just use parentheses, but haven't seen assignment like that

Reply:
using parenthesis makes the c++ compiler assume its a function and `SafeNumeric` has only an explicit constructor,, so these are the only ways im aware of